README.md

```diff
@@ -1,46 +1,96 @@
 # 🌏 Planet Read
 
-🐬 MySQL + 🐍 Flask story translation platform.  
+🐬 MySQL + 🐍 Flask story translation platform.
 
 Made with [starter-code-v2](https://github.com/uwblueprint/starter-code-v2), brought to you by the @uwblueprint/internal-tools team!
 
 # Getting Started
+
 ## Vault
+
 Get [started with vault](https://www.notion.so/uwblueprintexecs/Secret-Management-2d5b59ef0987415e93ec951ce05bf03e). To grab the project's secrets, run
+
 ```
 vault kv get -format=json kv/planet-read | python update_secret_files.py
 ```
+
 You should have three new files in your repo after this:
+
 - `.env`
 - `backend/.env`
 - `backend/python/firebaseServiceAccount.json`
+
 ## Prereqs
+
 Verify that you have docker and npx installed:
+
 ```
 docker info
 docker-compose --version
 npx -v
 ```
+
 # Build and Run
-Note: if you have already built the project before, run this first: 
+
+Note: if you have already built the project before, run this first:
+
 ```
 docker-compose down --volumes
 ```
 
-And run the project:
+This will take down the database and all it's data too.
+If you don't need to rebuild packages between switching branches, you probably don't _need_ `--volumes`.
+
+To run the project:
+
 ```
 docker-compose up --build
 ```
 
-Seed your database with the provided script. Before running, you will need to set environment variables for all of the user secrets, which can be found in the secrets channel.
+Insert test data into the database with the provided script. Your environment variables will need to be correctly configured in `.env`
+
 ```
-./seed_db.sh
+docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"
 ```
 
 If there are no tables in the DB, go into `/backend/python/app/models/__init__.py` and change the `erase_db_and_sync = False` to True, allow the hot reload to build, and change it back to `False`. Try to seed the database again.
```

Comment:
they can also run `flask db upgrade` to create the tables--should it be included in here as well (it is below in the Database Migrations section)?

Reply:
yep in the db migration section too!